.github/workflows/want-lgtm-all.yml

```diff
@@ -42,34 +41,13 @@ permissions:
 jobs:
   want-lgtm-all:
     if: |-
-      ${{ contains(fromJSON('["pull_request", "pull_request_review"]'), github.event_name) }}
+      contains(fromJSON('["pull_request", "pull_request_review"]'), github.event_name)
     runs-on: 'ubuntu-latest'
     steps:
       - id: 'want-lgtm'
-        name: 'Validate PR Body'
-        uses: 'actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea' # ratchet:actions/github-script@v7
-        with:
-          result-encoding: 'string'
-          script: |-
-            const { data: pullRequest } = await github.rest.pulls.get({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              pull_number: context.issue.number
-            });
-            const body = pullRequest.body || "";
-
-            // if PR body does not contain want_lgtm_all, exit gracefully
-            if (!body.toLowerCase().includes("want_lgtm=all")) {
-              core.info("PR body does not contain want_lgtm=all");
-              return "false";
-            }
-
-            return "true";
-
-      - id: 'validate-reviews'
         name: 'Validate Reviews'
         if: |-
-          ${{ steps.want-lgtm.outputs.result == 'true' }}
+          contains(github.event.pull_request.body, 'want_lgtm=all')
```

Comment:
I think you are dropping the `toLowerCase`, unless that contains function does that already for you.

Reply:
That function is not case sensitive FTW!

https://docs.github.com/en/actions/learn-github-actions/expressions#contains